setup.py

```diff
@@ -236,7 +236,7 @@ def rec(path, pkg_name):
         "numba.annotations": ["*.html"],
         # Various test data
         "numba.cuda.tests.cudadrv.data": ["*.ptx"],
-        "numba.hsa.tests.hsadrv": ["*.brig"],
+        "numba.roc.tests.hsadrv": ["*.brig"],
```

Comment:
I don't think we are still shipping any .brig files.  There is one unit test, but it references a file under `/opt/rocm'.

Reply:
Ah yes, thanks, I don't think any brigs are shipped now.